tests/resource_test.py

```diff
@@ -5,6 +5,7 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 import pytest
+from pykern.pkcollections import PKDict
```

Comment:
Move this import inside test_render_resource. We avoid all imports in py.test to ensure we can control module initialization order.